proto/hub-genesis/genesis.proto

```diff
@@ -12,4 +12,5 @@ message GenesisState {
   // params defines all the parameters of the module.
   Params params = 1 [ (gogoproto.nullable) = false ];
   State state = 2 [(gogoproto.nullable) = false];
+  repeated uint64 unacked_transfer_seq_nums = 3 [(gogoproto.nullable) = false];
```

Comment:
not really IMO
because you'd read and write a lot of them every time 
it's better as a map data structure (o(1) access)
we may assume chains can have a lot of genesis accounts (airdrop)